airflow/providers/neo4j/hooks/neo4j.py

```diff
@@ -44,8 +45,8 @@ def __init__(self, conn_id: str = default_conn_name, *args, **kwargs) -> None:
         self.neo4j_conn_id = conn_id
         self.connection = kwargs.pop("connection", None)
         self.client = None
-        self.extras = None
-        self.uri = None
+        self.extras: Dict = {}
+        self.uri: str = ""
```

Comment:
Same comment here if these really need to be hook instance attributes.

Reply:
changed.